packages/config/config/devices/0x021f/dmdp1.json

```diff
@@ -14,6 +14,28 @@
 		"max": "255.255"
 	},
 	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic_Set",
```

Comment:
The labels don't conform with our style guide: https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=association-groups

For example, `Basic_Set` should be `Basic Set` - apply to the others in a similar fashion.